src/Routing/Router.php

```diff
@@ -979,7 +982,39 @@ public static function scope($path, $params = [], $callback = null)
         if (is_array($params)) {
             $options = $params;
             unset($params['routeClass'], $params['extensions']);
+            $cache = (string)Hash::get($params, '_cache');
+            if (!empty($cache)) {
+                unset($params['_cache']);
+                static::$_collection = Cache::remember(
+                    $path,
+                    function () use ($path, $params, $options, $callback) {
+                        self::_buildScope($path, $params, $options, $callback);
+
+                        return static::$_collection;
```

Comment:
I'll check if serialize is disable and handle serialization here before saving into cache. Thanks for raising the flag.